src/package.json

```diff
@@ -13,7 +13,11 @@
     "template",
     "front-end"
   ],
-  "author": "",
+  "author": {
```

Comment:
not a big deal, but what's the advantage/reason for adding these additional fields for author? Isn't having a single line simpler?
https://docs.npmjs.com/files/package.json#people-fields-author-contributors